turborepo-tests/integration/tests/dry-json/monorepo-no-changes.t

```diff
@@ -2,13 +2,17 @@ Setup
   $ . ${TESTDIR}/../../../helpers/setup_integration_test.sh
 
 # Save JSON to tmp file so we don't need to keep re-running the build
-  $ ${TURBO} run build --dry=json --filter=main > tmpjson.log
+  $ ${TURBO} run build --dry=json --filter='[main]' > tmpjson.log
```

Comment:
I think I wrote these tests? but I am not sure if even this is valid usage of `--filter`

Reply:
I believe you wanted to write a filter for changes since main, but without the brackets it was just a filter for the non-existent `main` package.